_posts/2016-04-20-squabble-comments.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "How we added comments to our Jekyll site"
+date: 2016-04-20
+author: Kosta Harlan
+tags: jekyll
+summary: |
+  A peek under the hood at our website's commenting app, [Squabble](http://github.com/savaslabs/squabble).
+---
+[Avid readers of our blog](/2015/04/01/building-our-site.html) know that it's built using the amazing and lovely open-source static site generator known as [Jekyll](http://jekyllrb.com/).
+
+Jekyll is a great solution for our site for a site like ours because we get an always-on[^1], super fast, database-free site that's also capable of many of the things you'd typically associate with a database-driven site: tagging, content organization, advanced templating, etc. Best of all it's one less application for us to host and keep up-to-date with the latest security patches.
+
+However, as much as we like our static HTML, we also care about you, dear reader, and we wanted to provide an easy way for you to comment on and contribute to discussion on our site's content.
+
+This post provides an overview of how and why we implemented comments for our Jekyll site. If you find it interesting or helpful... leave a comment!
+
+## How most people solve the problem
+
+Most folks using static site generators include comments by outsourcing their site's comment section to [Disqus](https://disqus.com/).
+
+Disqus works like this: you sign up for their service and get a `<script>` embed code that you place in your page template. When the user visits that page, your browser will load the Disqus commenting widget. All comments are stored on Disqus' servers, the script loads any comments stored in Disqus for the particular URL that you are on.
```

Comment:
When the user visits that page, THEIR browser will load the Discus commenting widget?